src/containers/Home/index.jsx

```diff
@@ -25,13 +25,17 @@ const Home = () => {
     if (artworkTypeText && artworkTypeText !== artworkTypeTitle) {
       return false;
     }
-    // if (sourceFromDate && sourceFromDate > sourceUpdatedAt) {
-    //   return false;
-    // }
-    // if (sourceToDate && sourceToDate < sourceUpdatedAt) {
-    //   return false;
-    // }
-
+    if (sourceFromDate && new Date(sourceFromDate) > new Date(sourceUpdatedAt)) {
```

Comment:
This looks a bit off to me. Please take a look at this post to understand how to set a date to the beginning or end of day using JavaScript
https://javascript.plainenglish.io/how-to-get-the-start-and-the-end-of-a-day-in-javascript-b066c91ac488